src/gsCore/gsMath.h

```diff
@@ -326,7 +326,8 @@ template <typename T> int getSign(T val)
 template <typename T>
 inline typename util::make_unsigned<T>::type abs_diff(T a, T b)
 {
-    return a > b ? a - b : b - a;
+    typedef typename util::make_unsigned<T>::type unsignedT;
```

Comment:
not sure if this is really needed.
The Unittest runs without very fine.

Reply:
but it is ub

Reply:
ub?